### PR TITLE
UI localization not building when importing npm

### PR DESCRIPTION
**Pull request by Cagince**
fixed missing file reference error by adding a build step (`copy-json`) in order to copy json files in the production build. 
related to #279

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [develop]
 
+### Added
```
Comment (stonko1994) [inline CHANGELOG.md]: I would remove the added section. Even if we fixed it by adding something there is no value for this entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+- Additional build step to copy json files to production build
+
 ### Fixed
 - UI flickering back and forth on live streams after ads
+- i18n module missing files error
```
Comment (stonko1994) [inline CHANGELOG.md]:
```suggestion
- Build errors when using the npm package
```

File: CHANGELOG.md (hunk 1 above)
Comment (protyposis) [reply CHANGELOG.md]: Yes this is irrelevant to library users and it's not a new feature. The "fixed" entry is sufficient.

File: gulpfile.js [hunk 3]
```diff
@@ -198,7 +204,7 @@ gulp.task('build', gulp.series('clean', gulp.parallel('html', 'browserify', 'sas
 gulp.task('build-prod', gulp.series(function(callback) {
   production = true;
   callback();
-}, 'lint', 'build'));
+}, 'lint', 'build', 'copy-json'));
```
Comment (protyposis) [inline gulpfile.js]: I think this should be part of the `build` task, independent from the production flag.

File: gulpfile.js [hunk 4]
```diff
@@ -193,7 +199,7 @@ gulp.task('sass', function() {
 });
 
 // Builds the complete project from the sources into the target directory
-gulp.task('build', gulp.series('clean', gulp.parallel('html', 'browserify', 'sass')));
+gulp.task('build', gulp.series('clean', gulp.parallel('html', 'browserify', 'sass',  'copy-json')));
```
Comment (protyposis) [inline gulpfile.js]: Whitespacing!

**Pull request merged**